addons/io_hubs_addon/components/operators.py

```diff
@@ -433,6 +442,79 @@ def invoke(self, context, event):
         return wm.invoke_props_dialog(self, width=600)
 
 
+class GenerateNavMesh(Operator):
+    bl_idname = "object.generate_hubs_navmesh"
+    bl_label = "Generate Hubs Navigation Mesh"
+    bl_options = {'REGISTER', 'UNDO'}
+
+    def execute(self, context):
+        import bmesh
+
+        obj = bpy.context.active_object
```

Comment:
Would be great if we can generate a nav mesh from multiple selected objects instead of only the active one